Sources/App/Views/PublicPage.swift

```diff
@@ -275,6 +277,18 @@ class PublicPage {
         .empty
     }
     
+    /// Optional content that will be inserted just after the start of the body.
+    /// - Returns: An optional element, or group of elements.
+    func preBody() -> Node<HTML.BodyContext> {
```

Comment:
If I hadn't seen the naming discussing I'd think that either the name or the doc string got out of sync 🤪

Some alternatives that might work perhaps: `bodyOpening|Closing`, `opening|closingBody`, `leading|trailingBody`, `bodyLeading|Trailing`.

Reply:
I hate naming things 😂

Reply:
There's always caching if you need a break 🤣

Reply:
Me too, and this is a tricky one.

I don't hate `preBody` and `postBody`. They are not pre/post the body tag, but they are pre/post the body itself.